### DISCUSSION TITLE
Unfurl after space

### DISCUSSION THREAD

**Discussion by ndeville** [espanso/espanso#1306] [Feature requests and Ideas]
Just started using Espanso and love it so far! Thanks for building it!

One behaviour I'm missing from aText is the ability to unfurl snippets only after hitting `space`. 

This provides the ability to have closely related triggers like `;d` (for current date) and `;dt` (for current date & time) without the date unfurling after hitting `d` in `;dt`, making the latter unusable. 

I have read the full documentation and could not find a way around that. Perhaps I missed it?

**Comment by kpym** [comment]
I see two possibilities : 
-  use `word: true` option.
- or use `regex` like this `regex: ";d\\s"` 

**Note:** _This do not looks like "Feature requests and Ideas". This shoud be "Q&A"._
